### PR TITLE
code

### PR DESCRIPTION
**Pull request by liza-strykharchuk**
[DEMO LINK](https://liza-strykharchuk.github.io/react_movies-list-filter-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,33 +1,57 @@
...
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function getFilms(movies, querty1) {
+  let newMovies = [...movies];
```
Comment (mykhalenych) [inline src/App.jsx:8]: redundant filter already create copy

File: src/App.jsx [hunk 2]
```diff
@@ -1,33 +1,57 @@
...
+  const newQuerty = querty1.toLowerCase().trim();
+
+  if (newQuerty) {
+    newMovies = newMovies.filter(element => (
```
Comment (mykhalenych) [inline src/App.jsx:12]:
```suggestion
    return movies.filter(element => (
```

File: src/App.jsx [hunk 3]
```diff
@@ -1,33 +1,57 @@
...
+      element.title.toLowerCase().includes(newQuerty)
+      || element.description.toLowerCase().includes(newQuerty)
+    ));
+  }
+
+  return newMovies;
```
Comment (mykhalenych) [inline src/App.jsx:18]:
```suggestion
  return movies;
```

File: src/App.jsx [hunk 4]
```diff
@@ -1,33 +1,57 @@
...
+              Search movie
+            </label>
+
+            <div className="control">
+              <input
+                type="text"
+                id="search-query"
+                className="input"
+                placeholder="Type search word"
+                onChange={(querty2) => {
```
Comment (mykhalenych) [inline src/App.jsx:41]: why querty2?
create better naming

**Review by mykhalenych — CHANGES_REQUESTED** [review]

**Review by nazarmatsevych — APPROVED** [review]
Good work!

**Pull request closed without merging**